tests/ccgbugs/targ_lefttoright.nim

```diff
@@ -65,6 +65,16 @@ template test =
   var h = 1
   say h, h +=+ 1 # 1,2
 
+  proc mutateFirst(a: var C, b: C) =
+    a.i = 2
+    doAssert b.i == 1
+
+  var i = C(i: 1)
+  when nimvm: # XXX: doesn't work yet
+    discard
```

Comment:
this gave me an idea, maybe we should add a `doAssertKnownIssue` that'll work with inverted behaviour.

Reply:
I like it, I've come across multiple cases were that would have been useful.